core/Archive/ArchiveInvalidator.php

```diff
@@ -576,7 +577,10 @@ public function applyScheduledReArchiving()
                     continue;
                 }
 
-                $idSites = Site::getIdSitesFromIdSitesString($entry['idSites']);
+                $idSites = Access::doAsSuperUser(function () use ($entry) {
```

Comment:
would this invalidate things maybe for too many sites even where we shouldn't? Like if a user doesn't have access to these sites then we wouldn't want to invalidate them?

Reply:
in the context of a segment where idSites = 'all', then no matter what it means every site, correct? these are also in `applyScheduledReArchiving` where we are applying the saved scheduleRearchiving() calls, which means those could be called in different contexts and access levels, but will still save `'all'`. if it's possible that 'all' might not mean every site installed in a rearchiving context (rearchiving segment report, rearchiving plugin reports, rearchiving plugin entity reports), then maybe we should rethink this a bit more.

Reply:
I'm thinking `idsites=all` be mostly triggered by super users (eg activating a plugin or making a segment available for all sites). 

If someone who is not super user was to create something for multiple sites, then it should not appear in other sites.

Not sure if this helps?

Reply:
I'll just remove this, if it becomes a problem it can be fixed later.

Reply:
👍